en/docs/use-cases/streaming-usecase/exposing-stream-as-managed-api-in-service-catalogue.md

```diff
@@ -0,0 +1,42 @@
+# Async API Specification publishing into WSO2 API Manager Service Catalogue
+
+## Introduction
+
+This section provides an overview on how to export the Async API specification into the service catalogue during the Siddhi app deployment in the WSO2 SI.
+
+When the Siddhi application contains `@App:asyncAPI`, this specification will be exposed as APIs in the API Manager's service catalogue.
+
+## Before you begin
+
+Note: Please follow the steps in [Working with Async API View in Streaming Integrator Tooling]({{base_path}}/develop/streaming-apps/working-with-the-async-api-view) before you 
+continue on the following steps unless, the Siddhi application has a valid Async API definition.
+
+    1. Unzip API Manager server and in the <AM_HOME>/repository/conf/deployment.toml file, uncomment offset in the [server] section and set it to `5'.<br/>
+          [server]
+          offset=5
+    2. Unzip Streaming Integrator and enable `async.api.configs` in <SI_HOME>/conf/server/deployment.yaml and set `port` as `9448`
+          async.api.configs:
+            enabled: true
+            hostname: localhost
+            port: 9448
+            username: admin
+            password: admin
+    3. Start the Streaming Integrator, Streaming Integrator Tooling, and API Manager servers.
+
+## Exporting Async API spec to service catalogue in WSO2 API Manager from SI Server
```

Comment:
The current content under this topic are observations and not action items. Suggesting the subheadings to be reorganized.